vocabularies/csv/locations.csv

```diff
@@ -1,2430 +1,2430 @@
-mabb1,Schwarzman Building - Art Reference Room 300,SASB - Art Reference Rm 300,Research,,,ma,1130,false,mab,,true,,sierra-location
-mabb2,Schwarzman Building - Art & Architecture Room 300,SASB - Art & Architecture Rm 300,Research,,,ma,1110,false,mab,,true,,sierra-location
-mabb3,Schwarzman Building - Art & Architecture Desk Room 300,SASB - Art - Desk Rm 300,Research,,,ma,1110,false,mab,,true,,sierra-location
-mabm2,Schwarzman Building - Art & Architecture Room 300,SASB - Art & Architecture Rm 300,Research,,,ma,1110,false,mab,,true,,sierra-location
-mac,Schwarzman Building - Arents Collection Room 328,SASB - Arents Collection Rm 328,Research,,,ma,1109,false,mac,,true,arents-collection,sierra-location
-mac82,Schwarzman Building - Arents Collection Room 328,SASB - Arents Collection Rm 328,Research,,,ma,1109,false,mac,,true,,sierra-location
-macc2,Schwarzman Building - Arents Collection Room 328,SASB - Arents Collection Rm 328,Research,,,ma,1109,false,mac,,true,,sierra-location
-mae,Schwarzman Building - Berg Collection Room 320,SASB - Berg Collection Rm 320,Research,,,ma,1104,false,mae,,true,berg-collection-english-and-american-literature,sierra-location
-mae82,Schwarzman Building - Berg Collection Room 320,SASB - Berg Collection Rm 320,Research,,,ma,1104,false,mae,,true,,sierra-location
-maedd,SASB EDD Onsite,SASB EDD Onsite,Research,,,ma,,false,,,true,,sierra-location
-maee2,Schwarzman Building - Berg Collection Room 320,SASB - Berg Collection Rm 320,Research,,,ma,1104,false,mae,,true,,sierra-location
-maf,Schwarzman Building - Dorot Jewish Division Room 111,SASB - Dorot Jewish Division Rm 111,Research,Research,,ma,1103,false,maf,NJ,true,jewish-division,sierra-location
-maf82,Schwarzman Building - Dorot Jewish Division Room 111,SASB M1 - Dorot Jewish Division Rm 111,Research,,,ma,1103,true,mab;mag;map;maf;mal;mai;mala;malc;maln;malw;mal17,,true,,sierra-location
-maf88,Schwarzman Building - Dorot Jewish Division Room 111,SASB - Dorot Jewish Division Rm 111,Research,,,ma,1103,false,maf,,true,,sierra-location
+ag,Aguilar,Aguilar,Branch,,,ag,,false,,,true,,
+aga,Aguilar Adult,Aguilar Adult,Branch,,,ag,,false,,,true,,
+aga01,Aguilar Reference,Aguilar Reference,Branch,,,ag,,false,,,true,,
+aga03,Aguilar Closed Shelf Reference,Aguilar Closed Shelf Reference,Branch,,,ag,,false,,,true,,
+aga0f,Aguilar Fiction,Aguilar Fiction,Branch,,,ag,,false,,,true,,
+aga0l,Aguilar World Languages,Aguilar World Languages,Branch,,,ag,,false,,,true,,
+aga0n,Aguilar Non-Fiction,Aguilar Non-Fiction,Branch,,,ag,,false,,,true,,
+aga0v,Aguilar Non-Print Media,Aguilar Non-Print Media,Branch,,,ag,,false,,,true,,
+aga0w,Aguilar Center for Reading & Writing,Aguilar Center for Reading & Writing,Branch,,,ag,,false,,,true,,
+agar,Aguilar Adult Reference,Aguilar Adult Reference,Branch,,,ag,,false,,,true,,
+agj,Aguilar Children,Aguilar Children,Branch,,,ag,,false,,,true,,
+agj01,Aguilar Children’s Reference,Aguilar Children’s Reference,Branch,,,ag,,false,,,true,,

```

Comment:
Ah! Smart quotes! Please let's keep our quote characters in the lower ascii range

Reply:
i can replace them all with the regular single quote, it was just making the validate changes very noisey.